scipy/stats/tests/test_sampling.py

```diff
@@ -1005,10 +1005,8 @@ def test_inf_nan_domains(self, domain, err, msg):
         with pytest.raises(err, match=msg):
             NumericalInverseHermite(StandardNormal(), domain=domain)
 
-    @pytest.mark.filterwarnings('ignore::RuntimeWarning')
-    @pytest.mark.xslow
-    @pytest.mark.parametrize(("distname", "shapes"), distcont)
-    def test_basic_all_scipy_dists(self, distname, shapes):
+
```

Comment:
```suggestion
```

Reply:
If other tests pass, this can probably get committed with [skip ci] before merge to fix the lint issue.

Reply:
Just a small change, so I applied it. Hope you don't mind!

Reply:
Yup, that was the intent. Thanks!